velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -456,5 +456,24 @@ TEST_F(DateTimeFunctionsTest, quarterDate) {
   EXPECT_EQ(3, quarter("1954-08-08"));
 }
 
+TEST_F(DateTimeFunctionsTest, fromUnixTime) {
+  const auto fromUnixTime = [&](std::optional<int64_t> unixTime,
+                                std::optional<std::string> timeFormat) {
```

Comment:
Maybe add test for singe-arg case. In this case, timeFormat could be std::nullopt.

Reply:
As explained above, format is always contained in the plan.